### ISSUE TITLE
Could not find a declaration file for module '@noction/vue-bezier'

### ISSUE THREAD

**Issue by glutaminefree** [Noction/vue-bezier#5]
Got this error on `npm run build`  command in my project:

![image](https://github.com/Noction/vue-bezier/assets/2122419/87cab2cc-317d-4ced-91c4-b5a7e2e9b800)

```
Could not find a declaration file for module '@noction/vue-bezier'. 'path-to-project/node_modules/@noction/vue-bezier/dist/vue-bezier.js' implicitly has an 'any' type.
  There are types at 'path-to-project/node_modules/@noction/vue-bezier/dist/types/main.d.ts', but this result could not be resolved when respecting package.json "exports". The '@noction/vue-bezier' library may need to update its package.json or typings.ts(7016)
  ```

**Comment by 50rayn** [comment]
Hello @glutaminefree .
I've tried to reproduce this bug in a clear project with latest Vue3, Vite, VueBezier and TS version, and couldn't reproduce it.
Could you please provide more details?
We already encountered similar issue (like [this](https://github.com/Noction/vue-draggable-grid/issues/43)). 
Would be very helpfull if you provide more details. Perferctly a reproduction link on [stackblitz.com](https://stackblitz.com/).

**Comment by glutaminefree** [comment]
Can't reproduce it on [stackblitz.com](https://stackblitz.com/). May be trouble in my nodejs version?

node - v18.19.0
npm - 9.8.1

**Comment by 50rayn** [comment]
I would try to clear the cache and reinstall + check the installed version.

1. `rm -rf node_modules`
2. `npm cache clean --force`
3. `npm i`
4. go to the node_modules/@noction/vue-bezier
5. check version from package.json - to ensure that is installed the latest version

**Comment by glutaminefree** [comment]
This not helped. Version of @noction/vue-bezier in package.json is 1.11.2 and same in node_modules.
Error not reproduced in stackblitz, but did it in docker with node:18.

**Comment by tomastan** [comment]
Same here.  Freshly installed. Get this notice in VScode.

**Comment by 50rayn** [comment]
Fix available in v1.12.0.
